deployments/kustomize/base/WORDPRESS_CONFIG_EXTRA

```diff
@@ -1,4 +1,5 @@
 /* Multisite */
+require_once dirname(__DIR__) . '/html/vendor/autoload.php';
 define( 'WP_ALLOW_MULTISITE', true );
```

Comment:
why require just once?

Reply:
Good question. My understanding is that you should use require_once when the content of the files could produce an error by being included multiple times. Consider the simple case where the required files contain functions, which cannot be re-declared.